src/db/queries.js

```
@@ -24,8 +12,40 @@ async function getUserClasses(userId) {
   return db`SELECT * FROM classes WHERE user_id = ${userId}`
 }
 
+
+//add new class
+async function createClass(name, professor, details, userId) {
+  try {
+    const result = await db`
+      INSERT INTO classes (name, professor, details, user_id)
+      VALUES (${name}, ${professor}, ${details}, ${userId})
+      RETURNING *;
+    `;
+    return result;
+  } catch (error) {
+    console.error('Error creating class:', error);
```

In the future, let's return a good error response